xml/obs/systemsmanagement:Agama:Staging.xml

```diff
@@ -0,0 +1,15 @@
+<openQA
+    project_pattern="systemsmanagement:Agama:Staging"
+    dist_path="obspublish-other::openqa/systemsmanagement:Agama:Staging/"
+    version="agama-0.9"
+    archs="x86_64 aarch64">
+    <flavor name="ALP" folder="images/x86_64/agama-live:ALP" distri="alp">
+        <hdd filemask="agama-live.*-ALP-Build.*iso$"/>
+    </flavor>
+    <flavor name="default" folder="images/x86_64/agama-live:default" distri="alp">
```

Comment:
the flavor should not be fixed to one architecture, see how it is done in other places:
https://github.com/os-autoinst/openqa-trigger-from-obs/blob/master/xml/obs/SUSE%3AALP%3AMicro.xml#L6

Reply:
why the `distri="alp"` ? actually Agama is not related with ALP exclusively.

Reply:
I just following the distri name of https://openqa.opensuse.org/group_overview/96, so which name you prefer it ?

Reply:
> I just following the distri name of https://openqa.opensuse.org/group_overview/96, so which name you prefer it ?

I just following the distri name of https://openqa.opensuse.org/group_overview/96, so which name you prefer it ?

Reply:
> why the `distri="alp"` ? actually Agama is not related with ALP exclusively.

I just following the distri name of https://openqa.opensuse.org/group_overview/96, so which name you prefer it ?

Reply:
@jknphy ?

Reply:
doesn't look mandatory, we can omit it I guess, right?